modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerTest.java

```diff
@@ -14,10 +16,12 @@
 import static org.mockserver.model.HttpResponse.response;
 import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
 
+@Slf4j
 public class MockServerContainerTest {
 
     @ClassRule
-    public static MockServerContainer mockServer = new MockServerContainer();
+    public static MockServerContainer mockServer = new MockServerContainer(MockServerClient.class.getPackage().getImplementationVersion())
+        .withLogConsumer(new Slf4jLogConsumer(log));
```

Comment:
could you please remove the log consumer? Otherwise, it might get too noisy.

Reply:
Je n'ais pas encore compris